spec/lib/site_setting_extension_spec.rb

```diff
@@ -853,4 +853,36 @@ def self.translate_names?
       end
     end
   end
+
+  describe "_map extension for list settings" do
```

Comment:
Just a thought but `map` is commonly used to refer to key/value data types in programming languages. I wonder if we should go with `_list` here to make it clear an array is returned.

Reply:
I hear you, but I originally chose `_map` since the first one of these I did was doing `.map(&:to_i)` on the array, and I want to keep it consistent otherwise I have to go back and change the other one everywhere :sweat: